google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/scrape/metadata_values_converter.py

```diff
@@ -0,0 +1,69 @@
+from dateutil.relativedelta import relativedelta
+
+
+class MetadataValuesConverter:
+    '''
+    Duration-related values in Kafka, such as retention time,
+    are stored in ms, which might not always be human readable.
+    The same goes for space values, that Kafka stores in bytes.
+    This class converts those time and space values to
+    human-readable format.
+    '''
+
+    @staticmethod
+    def get_human_readable_duration_value(duration):
```

Comment:
I'd make it explicit in the param name that this is milliseconds. `duration_ms` perhaps?